src/consumer/component/dependencies/files-dependency-builder/precinct/index.ts

```diff
@@ -107,6 +111,8 @@ function precinct(content, options) {
       theDetective = detectiveVue;
       break;
     default:
+      detector = detectorHook.getDetector(type);
```

Comment:
what if the detector was not found?
I'd check for it and do nothing. similar as it how was before. there are files that should not be processed